### PR TITLE
docs: editing and translation for the data import doc #827

### LINKED ISSUE #827: editing and translation for the data import doc
It consists of two tasks

- [ ] proofreading the Chinese version doc `docs/cn/data_import.md`
- [ ] translating this Chinese version to English, and named as `docs/en/data_import.md`

### PR DESCRIPTION
**Pull request by yangcol**
editing and translation for the data import doc #827

add file docs/en/data_import.md

### CODE REVIEW

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by yangcol** [comment]
HAH

File: docs/en/data_import.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+`insert` mode is suitable for almost every situation, and we are keeping optimizing it.
+
+`bulk load` mode is only suitable for empty table loading with no reading and writing before the loading process stopped. 
+
+The `bulk load` mode is more efficient and is available for multiple local files with csv format. 
+
+## Bulk Load
+
+`bulk load` can be used for large-scale data importing, and it's fast.
+But be careful, you have to make sure the target table emptly before the data is loaded, and don't do some read or write operation during data importing.
```
Comment (aceforeverd) [inline docs/en/data_import.md]: typo: emptly -> empty

File: docs/en/data_import.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+`bulk load` is driven by two components. The first one is the importer, it's the import tool itself. The other one is the loader, it's the tablet server of openmldb.
+
+Usually, there are three steps for importing data.
+Firstly, the importer sends the primary data to the loader.
+Secondly, the loader parses the data and does insert operation, which includes locating the segment and inserting data into two level skip list of the segment.
+Thirdly, binlog is written for synchronizing replica data. 
+
+`bulk load` makes the data inserting process executing first. Then the importer generates all the segments, and sends to the loader directly. The loader can rebuild all the segments in linear time and complete the whole data importing process.
+
+We call the data `data region` and binlog is part of `data region`. And multiple segment are called `index region`. `data region` is sent in batches during the importer scans data source. `index region` is sent after all data are processed, and it may be sent in batches if the size exceeds the limit of the loader's rpc. When the last `index region` part is sent, the whole imporing process completes.
```
Comment (aceforeverd) [inline docs/en/data_import.md]: typo: imporing -> import

File: docs/en/data_import.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Currently, import tool can only be executed in standalone mode. Large scale data will occupy more resource, so some operations can become time-consuming.
+
+If oom triggers, you have to increase the `-Xmx` value. 
+If rpc send fails, you have to increase `rpc_write_timeout` value, and if rpc respond timeout, please increate `rpc_read_timeout` value properly.
+
+#### Error Handling
+
+Importing process fails when `bulk load success` not printed by the importer, or when the importer quit or when failed message is printed by the importer.
+
+Because of the complexity of importing failure, the correct way to handle the eror is to drop the table with failure message and rebuild table, and then import again.
```
Comment (aceforeverd) [inline docs/en/data_import.md]: typo: eror -> error

File: docs/en/data_import.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Table is not required to be existing because the importer can create a table. 
+
+But be careful, importing data into a non-empty table will decrease the import efficient severely.
+
+#### Configuration Adjustment
+
+Currently, import tool can only be executed in standalone mode. Large scale data will occupy more resource, so some operations can become time-consuming.
+
+If oom triggers, you have to increase the `-Xmx` value. 
+If rpc send fails, you have to increase `rpc_write_timeout` value, and if rpc respond timeout, please increate `rpc_read_timeout` value properly.
```
Comment (aceforeverd) [inline docs/en/data_import.md]: typo: increate -> increase

**Comment by lumianph** [comment]
@yangcol can you help address those comments, please? Thanks.

**Comment by aceforeverd** [comment]
@yangcol thx for contributing.

**Review by lumianph — APPROVED** [review]

**Pull request merged**